src/lwc/utilCommon/utilCommon.js

```diff
@@ -552,6 +552,38 @@ const replaceLastInstanceOfWith = (subject, toRemove, replacement) => {
     return subject && subject.replace(new RegExp(toRemove + '$'), replacement);
 }
 
+
+/**
+ * @description Converts field describe info into a object that is easily accessible from the front end
+ * Ignore errors to allow the UI to simply not render the layout-item if the field info doesn't exist
+ * (i.e, the field isn't accessible).
+ */
+const extractFieldInfo = (fieldInfos, fldApiName) => {
```

Comment:
@psadie-sfdo , @kenlewis-sfdo , @vmartinov-sforg: FYI -- I added two new common methods to extract field infos and build field describe. Please let me know if there is any concern.

Reply:
No concern from me - thank you for the heads-up!